python/ray/_private/services.py

```diff
@@ -1735,6 +1731,20 @@ def get_ray_jars_dir():
     return os.path.abspath(os.path.join(current_dir, "jars"))
 
 
+def get_ray_native_library_dir():
+    """Return a directory where all ray-related native libraries and
+    their dependencies locate."""
+    current_dir = RAY_PATH
+    native_library_dir = os.path.abspath(os.path.join(current_dir, "cpp/lib"))
+    if not os.path.exists(native_library_dir):
+        raise RuntimeError(
+            "Ray native libraries is not packaged into ray. "
+            "Please install ray with option [cpp] "
+            "(pip install ray[cpp])"
```

Comment:
```suggestion
            "(pip install \"ray[cpp]\")"
``` for compatibility with `zsh`, the default terminal on newer Macs